client/src/pages/CreateAccount/CreateAccount.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from './CreateAccount.module.scss';
+import { useState } from 'react';
+import { IconButton, Typography, AppBar, Toolbar } from '@mui/material';
+import { ArrowBackIos } from '@mui/icons-material';
+import { CustomButton, PasswordField } from '@/components';
+
+export default function CreateAccount(): JSX.Element {
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  const handleClick = () => {
+    // TODO: This button should pass in the password and finish creating the user
+    // For now, it simply prints the values
+    console.log(password);
+    console.log(confirmPassword);
+  };
+
+  return (
+    <div className={styles['Create Account']}>
+      <AppBar
+        position="static"
+        sx={{ background: 'transparent', boxShadow: 'none', color: 'black' }}
```

Comment:
You should have the styling in the .scss file and import it from there. 

I see that you are styling an MUI component, Jordan shared this with us that shows how to do so: https://medium.com/@savindiwaduge7/styling-with-mui5-sass-in-a-design-system-bb617b920d13
Although i dont think it makes a huge difference.

Reply:
I've moved the styling to the separate `CreateAccount.module.scss` file. For the AppBar component, I had to add the `!important` rule for the styling to work properly. If there are better ways to fix this, let me know!